OgreMain/src/OgreCamera.cpp

```diff
@@ -174,7 +171,7 @@ namespace Ogre {
 
         Quaternion targetWorldOrientation;
 
-        if( mYawFixed )
```

Comment:
I would prefer not to do these whitespace changes manually. They are trivial to do with clang-format. I just refrained to apply them as such commit introduces lots of noise in the code (think of open pull requests) without adding much.

Reply:
is there a .clang-format ready to use somewhere or we can use llvm with 4 space of indentation ?

Reply:
added the clang-format I use to #2584. But still, I do not think we should apply it to existing code just like that..

Reply:
applied a slightly similar format than your, I think it looks good